packages/js/src/ui/components/Footer.tsx

```diff
@@ -0,0 +1,10 @@
+import { Novu } from '../icons';
+
+export const Footer = () => {
+  return (
+    <div class="nt-flex nt-justify-center nt-items-center nt-gap-1 nt-mt-auto nt-py-3 nt-text-foreground-alpha-200">
```

Comment:
Eslint should complain here for not using `style()`. However, it is the single place we don't want to add descriptors, so let's add an exception to the eslint rule for these two elements.